benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -122,6 +122,45 @@ void GraphicsBenchmarkApp::InitKnobs()
     pFullscreenQuadsSingleRenderpass->SetDisplayName("Single Renderpass");
     pFullscreenQuadsSingleRenderpass->SetFlagDescription("Render all fullscreen quads (see --fullscreen-quads-count) in a single renderpass.");
     pFullscreenQuadsSingleRenderpass->SetIndent(1);
+
+    GetKnobManager().InitKnob(&pRenderOffscreen, "enable-offscreen-render", false);
+    pRenderOffscreen->SetDisplayName("Offscreen render");
+    pRenderOffscreen->SetFlagDescription("Enable render to a offscreen/non-swapchain framebuffer.");
```

Comment:
nit: since "render" is a verb, I think the current knob values sound slightly awkward.
Perhaps we can consider alternatives:
knob name: `enable-offscreen`, `enable-offscreen-rendering`, `enable-render-offscreen`
display name: "Render offscreen", "Offscreen rendering"
description: "Enable rendering to an offscreen/non-swapchain framebuffer."
Perhaps the description should also reference what happens based on whether offscreen-blit-to-swaphchain is true/false

Reply:
Done.

`offscreen-blit-to-swaphchain` is on by default so we can see the rendering result, it does not really affect measurement since the timestamp range excludes it. It's there in case we want to turn it off for some reason.